app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -129,11 +129,21 @@ const MakeDecision = ({ data }) => {
         },
       }));
     }
+    let comment = '';
+    if (itemData.issue.comment === modalState.source.issue.comment) {
```

Comment:
Are you sure `itemData.issue` always exists? The same question about `modalState.source.issue`

Reply:
`itemData` contain `issue` from props data and `modalState` inherit `issue` from props data or changing in user working process

Reply:
@chivekrodis 
I see. But are you sure that `data.items` or `data.items[0]` always has the property `issue`? For example, the `launchNumber` property may not be. And if `issue` does not exist it will cause the error.

Reply:
modal has validation `validateEditDefect` in case if item have no `issue` and modal window wouldn't open